### DISCUSSION TITLE
[Projects Beta] Have labels without converting to issue

### DISCUSSION THREAD

**Discussion by Mineinjava** [community/community#15380] [Projects and Issues]
In the new Projects beta, projects can be tied to a user or organization instead of a repository. However, to label a task in the project board or list, it must first be converted to an issue inside a repository.

This can cause confusion when a project encompasses multiple repositories, and a task relates to multiple of these repositories. 

Something like milestones within the projects could also be useful here, allowing users to set smaller goals. This would be especially useful to a "master project" where smaller projects and goals are delegated to groups or individuals. Milestones could also label tasks that contribute to the milestone.

**Comment by dkstrong** [comment]
I also would find this useful. I often have things I want to add to my project. But they need to be triaged/research/prepared some before I actually know what repo it is associated with. Having labels available would be great for adding categorization/organization.

**Comment by CosmoCreeper** [comment]
Yeah, I don't want to have my project stuff public so I don't add as an issue but if I don't, I can't add labels. Despite the integration benefits of GitHub Projects, this is the one thing stopping me from using it as Trello already has labels.
